TeamCode/src/main/java/org/firstinspires/ftc/teamcode/rework/TestOpModes/AutoPathTest.java

```diff
@@ -1,4 +1,4 @@
-package org.firstinspires.ftc.teamcode.rework.Tests;
+package org.firstinspires.ftc.teamcode.rework.TestOpModes;
```

Comment:
Rather than having tests in src/main, moving them to src/tests could be a good idea

Reply:
Maybe we should move testopmodes... we can do it later though, I just want to get this pulled first.